SudoRules-V20.1-config.clp

```diff
@@ -107,7 +107,8 @@
 
 
 
-
+(bind ?override ?override)  ; 'declare' override so the if statement succeeds
+(if ?override then (eval ?override) else
```

Comment:
This is the magic that lets us replace the user config with a test override.  We set the ?override variable to itself on the line above to ensure that it exists.  It will emit an error message but otherwise work as expected.  Without line 110, this line will fail if ?override is not defined.